charts/kubewarden-defaults/templates/policyserver-default.yaml

```diff
@@ -22,6 +24,7 @@ spec:
     {{- range $key, $value := .Values.policyServer.annotations }}
       {{ $key | quote }}: {{ $value | quote }}
     {{- end }}
+    {{- include "kubewarden-defaults.annotations" . | nindent 4 }}
```

Comment:
same as https://github.com/kubewarden/helm-charts/pull/178/files#r1060752392 here.

Reply:
removed in https://github.com/kubewarden/helm-charts/pull/178/commits/1a2faf835462d99e0f225c1a51c160357933d731, use the already present `Values.policyServer.annotations` for policy server